pkg/operator/k8sutil/prometheus_test.go

```diff
@@ -0,0 +1,23 @@
+package k8sutil
+
+import (
+	"os"
+	"path"
+	"testing"
+
+	"github.com/stretchr/testify/assert"
+)
+
+func TestGetServiceMonitor(t *testing.T) {
+	gopath := os.Getenv("GOPATH")
+	filePath := path.Join(gopath, "src/github.com/rook/rook/cluster/examples/kubernetes/ceph/monitoring/service-monitor.yaml")
+	_, err := GetServiceMonitor(filePath)
```

Comment:
It would be good to add some asserts to verify the properties of the service monitor that is loaded. Same for the rules below.

Reply:
ack